packages/node-utils/src/implementations/CryptoUtils.ts

```diff
@@ -661,4 +673,175 @@ export class CryptoUtils implements ICryptoUtils {
     }
     return Buffer.from(ethers.getBytes(hex));
   }
+
+  public parseRawP256PublicKey(
+    publicKey: P256PublicKey,
+  ): Result<
+    { x: P256PublicKeyPointX; y: P256PublicKeyPointY },
+    InvalidParametersError
+  > {
+    // Convert hex string public key to Uint8Array
+    const pubKeyView = this.hexToUint8Array(publicKey);
+
+    // Public Key Header Bytes
+    const headerByte = pubKeyView[0];
+
+    // Second value tells you the length of the rest of the data array
+    const keyLength = pubKeyView[1];
+
+    // Third value tells you the type of the next value which MUST be an integer (0x02) if this is a signature array
+    const metadataIndicatorByte = pubKeyView[2];
+    // Third byte MUST be equal to 48 if this is a legitimate public key array
+    if (metadataIndicatorByte !== 48) {
+      return err(new InvalidParametersError("This is not a public key array"));
+    }
+
+    // Forth Value is the length of the public key metadata
+    const metadataLength = pubKeyView[3];
+
+    // Slice out the metadata and print it
+    // this metadata is a SEQUENCE OF containing the description of the key type (i.e. ecPublickey for P-256)
+    // Really we should parse this and make sure the public key was generated for the curve we are expecting
+    // i.e. P256 (the object identifier for this curve is: 1.2.840.10045.3.1.7 https://oid-rep.orange-labs.fr/get/1.2.840.10045.3.1.7)
+    const metadataUint8Array = pubKeyView.slice(4, 4 + metadataLength);
+    const metadataString = metadataUint8Array.reduce(
+      (t, x) => t + x.toString(16).padStart(2, "0"),
+      "",
+    );
+
+    const publicKeyIndicatorByte = pubKeyView[4 + metadataLength];
+    // This byte MUST be equal to 2 if this is a legitimate signature array
+    if (publicKeyIndicatorByte !== 3) {
+      return err(new InvalidParametersError("This is not a public key array"));
+    }
+
+    // Now get the length of the s value of the signature (r,s)
+    const pubKeyLength = pubKeyView[4 + metadataLength + 1];
+
+    // Slice out the s value and print it
+    const startingByte = 4 + metadataLength + 2;
+    const endingByte = startingByte + pubKeyLength;
+    const publicKeyUint8Array = pubKeyView.slice(startingByte, endingByte);
+    const publicKeyString = publicKeyUint8Array.reduce(
+      (t, x) => t + x.toString(16).padStart(2, "0"),
+      "",
+    );
+    const qx = publicKeyString.slice(
+      publicKeyString.length - 128,
+      publicKeyString.length - 64,
+    );
+    const qy = publicKeyString.slice(-64);
+
+    return ok({
+      x: P256PublicKeyPointX(`0x${qx}`),
+      y: P256PublicKeyPointY(`0x${qy}`),
+    });
+  }
+
+  // returns a 64-byte ArrayBuffer containing r and s concatenated together
+  public parseRawP256Signature(
```

Comment:
Turns out this method isn't needed, I have this already baked into SimpleWebAuthn library.